models/example/customers.sql

```diff
@@ -0,0 +1,50 @@
+{{ 
+    config(
+        materilized='view')
+}}
```

Comment:
because you have the same config set in the schema.yml - this is redundant. also, view is the default materialization, so unless you're overriding another config you actually don't need to specify it at all 👍🏻